feature/signupflow/entercode/src/main/java/com/epmedu/animeal/signup/entercode/presentation/viewmodel/EnterCodeViewModel.kt

```diff
@@ -26,8 +32,21 @@ internal class EnterCodeViewModel @Inject constructor(
         viewModelScope.launch { launchCodeValidation() }
     }
 
+    private var lastCode: List<Int?> = emptyCode()
+
+    private val confirmCodeRequestHandler = object : AuthRequestHandler {
```

Comment:
Let's create AuthRequestHandler in the usecases as you did in other modules

Reply:
Moved all AuthRequestHandlers to use cases, please have a look.